topics/handlebars/index.md

```diff
@@ -10,4 +10,4 @@ related: mustache-js, ejs-templates, pug, nunjucks, jinja2, liquid, javascript,
 released: February 15, 2011
 github_url: https://github.com/handlebars-lang
 ---
-**Handlebars** is a simple templating language. It uses a template and an input object to generate HTML or other text formats. Handlebars templates look like regular text with embedded Handlebars expressions. Handlebars is largely compatible with Mustache templates. In most cases it is possible to swap out Mustache with Handlebars and continue using your current templates. A handlebars expression is a `{{`, some contents, followed by a `}}`. When the template is executed, these expressions are replaced with values from an input object.
+**Handlebars** is a simple templating language. It uses a template and an input object to generate HTML or other text formats. Handlebars templates look like regular text with embedded Handlebars expressions. Handlebars is largely compatible with Mustache templates. In most cases it is possible to swap out Mustache with Handlebars and continue using your current templates. A handlebars expression is a "{{", some contents, followed by a "}}". When the template is executed, these expressions are replaced with values from an input object.

```

Comment:
changing from backticks to quotes fixed [liquid warning](https://github.com/github/explore/actions/runs/11299945094/job/31431925940#step:6:82)

Reply:
@@ -10,4 +10,4 @@ related: mustache-js, ejs-templates, pug, nunjucks, jinja2, liquid, javascript,
released: February 15, 2011
github_url: https://github.com/handlebars-lang
---
**Handlebars** is a simple templating language. It uses a template and an input object to generate HTML or other text formats. Handlebars templates look like regular text with embedded Handlebars expressions. Handlebars is largely compatible with Mustache templates. In most cases it is possible to swap out Mustache with Handlebars and continue using your current templates. A handlebars expression is a `{{`, some contents, followed by a `}}`. When the template is executed, these expressions are replaced with values from an input object.
**Handlebars** is a simple templating language. It uses a template and an input object to generate HTML or other text formats. Handlebars templates look like regular text with embedded Handlebars expressions. Handlebars is largely compatible with Mustache templates. In most cases it is possible to swap out Mustache with Handlebars and continue using your current templates. A handlebars expression is a "{{", some contents, followed by a "}}". When the template is executed, these expressions are replaced with values from an input object.